backend/pkg/boards/events.go

```diff
@@ -0,0 +1,55 @@
+package boards
+
+import (
+	"github.com/HyperloopUPV-H8/h9-backend/pkg/abstraction"
+)
+
+type AckNot struct {
+	ID abstraction.BoardEvent // AckId
+}
+
+func (ack *AckNot) Event() abstraction.BoardEvent {
+	return ack.ID
+}
+
+type DownloadNot struct {
+	ID abstraction.BoardEvent // DownloadId
+}
+
+func (download *DownloadNot) Event() abstraction.BoardEvent {
+	return download.ID
+}
+
+type UploadNot struct {
+	ID   abstraction.BoardEvent // UploadId
+	Data []byte
+}
+
+func (upload *UploadNot) Event() abstraction.BoardEvent {
+	return upload.ID
+}
+
+type BlcuPing struct {
+	ID abstraction.TransportEvent // BlcuOrderId
+}
+
+func (blcuPing *BlcuPing) Event() abstraction.TransportEvent {
+	return blcuPing.ID
+}
+
+type BoardPush struct {
```

Comment:
Make different structs: `DownloadSuccess`, `UploadSuccess` `DownloadFailure` and `UploadFailure` (feel free to change it so it doesn't get too verbose)